README.md

```diff
@@ -1,33 +1,149 @@
 # CZG-Keremony
 
+Proto-danksharding requires a new cryptographic scheme: KZG Commitments.
+This client was created to participate in KZG Ceremony for Proto-danksharding (aka [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844)) in Ethereum.
+
+For more information about KGZ Ceremony, please read [Announcing the KZG Ceremony](https://blog.ethereum.org/2023/01/16/announcing-kzg-ceremony) at Ethereum foundation blog.
+
+## Which BLS library did the CZG-Keremony client use?
+
+The BLS library used in the CZG-Keremony client is the [noble-curves](https://github.com/paulmillr/noble-curves) library.
+
+
+## Why do we make a client with JavaScript?
+
+JavaScript is a versatile language that can run on a wide range of platforms and devices. While there are clients available for other programming languages such as Go, Rust, Java, and C++, there are currently no clients written in JavaScript. As a result, a client has been developed in JavaScript to add to the diversity of clients available. Having clients written in multiple programming languages can help decentralize the ceremony and make it more robust and secure.
+
+## Dependencies
+
+1. Install `Node.js v18+` and `npm`. 
+2. Run `npm install`.
+
+## How to use CZG-Keremony client?
+
+There are two methods for using the CZG Keremony client to participate in the KZG Ceremony. One option involves contributing through the command line interface (CLI), while the other involves using an interactive prompt.
+
+To participate in ceremony, you must have at least one that satisfies the following:
+- Ethereum - An Ethereum address is required to have sent at least 3 transactions before Jan. 13, 2023 (block number 16,394,155)
+- GitHub - A GitHub account that has a commit dated before 1 August 2022 00:00 UTC.
+
+### 1. Command Line Interface (CLI)
```

Comment:
I think it will be better to provide only interactive option.
And also we should provide the guide by using docker image, which will be more useful to whom don't want to install node.js env.

Reply:
I'll improve it a little more and add it later.